app/assets/javascripts/browse-columns.js

```diff
@@ -272,6 +286,28 @@
       return out;
     },
 
+    getRelatedTopicsData: function(state){
+      var data = this.sectionCache('related', state.slug);
+      var url = "/api/content" + state.path;
+
+      var out = new $.Deferred()
+      if(typeof state.detailedGuideData !== 'undefined'){
+        out.resolve(state.detailedGuideData);
+      }else if(typeof data !== 'undefined'){
```

Comment:
really picky, sorry: spacing after } here

Only commenting because it's so neat elsewhere.